docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -102,11 +106,148 @@ \section{Critical Assumptions}
 		& \multicolumn{1}{|c|}{\textbf{Recommended Action}}
 		& \multicolumn{1}{|c|}{\textbf{SR}}
 		& \multicolumn{1}{|c|}{\textbf{Ref.}}  \\
-		\hline
+        \hline
+        Authenti\-cation
+        & User is unable to login
+        & User cannot access their account and application features
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item User entered incorrect credentials
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Allow users to reset their password
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item SR1
+        \end{enumerate}
+        & HA1 \\
+        \cline{2-7}
+        & User's account is hacked by a third-party
+        & User's data can be read and modified by third-party
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Third-party gains access to the  user's account without their permission
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Refer to HA1-a
+            \item Limit the number of login attempts a person has for a given time period
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item SR1
+            \item SR2
+        \end{enumerate}
+        & HA2 \\
+        \hline
+        Purchase Data Input
+        & Application improperly reads user's inputted receipt
+        & Inaccurate data is stored to the database affecting application analytics and suggestions
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item OCR interprets certain characters incorrectly from the input
+            \item OCR is unable to read certain words or characters from the input
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Utilize natural language processing to clean and correct output produced by the OCR
```

Comment:
I'm pretty sure it's fine. All of the Recommended Actions are basically solutions to the Failure Modes. Even past examples have Recommended Actions that seem to be solutions:

![image](https://github.com/r-yeh/grocery-spending-tracker/assets/67233377/5b76ef78-bbfa-4078-b3eb-dc74196a86b1)

It's not saying how the natural language processing is going to clean it up, it just says that it will be used to help.